src/model/data_access_layer/groups/groups_reader.py

```diff
@@ -0,0 +1,12 @@
+from src.model.tables import Groups
+
+
+class GroupsReader:
+
+    def __init__(self, alchemist):
+        self.session = alchemist.session
+
+    def is_group_registered(self,
+                            chat_id: str) -> bool:
+        group = self.session.query(Groups).filter(Groups.id == chat_id).first()
```

Comment:
Please check is there some method like `.get()` . Lot of ORM provide it like shortcut for `.filter().frst()`